airgun/widgets.py

```diff
@@ -727,8 +727,9 @@ class SatFlashMessages(FlashMessages):
 
     """
 
-    ROOT = '//div[@class="toast-notifications-list-pf"]'
-    MSG_LOCATOR = f'{ROOT}/div[contains(@class, "alert")]'
+    ROOT = '//div[@class="pf-c-alert pf-m-success foreman-toast" or \
+        data-ouia-component-type="PF4/Alert"]'
+    MSG_LOCATOR = f'{ROOT}/div[contains(@class, "pf-c-alert__title")]'
```

Comment:
If possible can you run some other tests to ensure that this is not affecting other tests as this is common code seems?

Reply:
based on the earlier code, we used to get the description and span tag from  https://github.com/SatelliteQE/airgun/pull/661/files#diff-b6d36691d2c9b246c1a2f7fff90473fe0098b4a6846694417e9acc246e35ea30R706 was used to get the description, we may not be required now. as there is no span tag inside the description class. Ping me if you find above statement not clear to you.

Reply:
OK

as per https://github.com/SatelliteQE/robottelo/pull/9301 its used in SyncPlan tests,  I will search for others.

Reply:
This code is across the project https://github.com/SatelliteQE/airgun/blob/4ecf913c976845f4cf461b067b0cf6264eee865a/airgun/views/common.py#L41 -> common code and then you can see that reference here https://github.com/SatelliteQE/airgun/search?q=flash

Reply:
OK, thanks for that link, I understand now! 
So I think I should just look for one or two tests that are not related to my components that call those entities files.

e.g.  tests/foreman/ui/test_activationkey.py -k test_positive_add_host_collection_non_admin
as per https://github.com/SatelliteQE/robottelo/pull/9298#issuecomment-1012998601